### PR TITLE
Feature/docs 2302 api attack surface

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/scanner.md [hunk 1]
```diff
@@ -118,3 +118,13 @@ To return to the default settings, use an empty value or enter `0`.
 ![Setting domain RPS](../images/user-guides/scanner/set-rps-for-domain.png)
 
 If multiple domains are associated with the same IP address, the speed of requests to this IP address will not exceed the limits for the IP address. If multiple IP addresses are associated with one domain, then the total speed of requests to these IP addresses within this domain will not exceed the limits for the domain.
+
+## Preventing scanner from blocking
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/scanner.md:122]: scanner -> Scanner. the header sounds like scanner blocks smth. try another one

File: docs/latest/about-wallarm/api-leaks.md [hunk 2]
```diff
@@ -29,5 +27,44 @@ The **API Leaks** section provides rich visual representation for your current s
 
 ## Access API Leaks
 
-* For mitigation of the leaked API tokens threat, Wallarm [node(s)](../user-guides/nodes/nodes.md) should be deployed.
-* By default, the API Leaks module is disabled. To get access to the module, please send a request to [Wallarm technical support](mailto:support@wallarm.com).
+By default, the API Leaks module is disabled. To get access to the module, please send a request to [Wallarm technical support](mailto:support@wallarm.com). Note that for mitigation of the leaked API tokens threat, Wallarm [node(s)](../user-guides/nodes/nodes.md) should be deployed.
+
+When module is enabled, in Wallarm console, use **API Leaks** section to work with it:
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-leaks.md:32]: we have discussed several times to delete it

File: docs/latest/about-wallarm/api-leaks.md [hunk 3]
```diff
@@ -29,5 +27,44 @@ The **API Leaks** section provides rich visual representation for your current s
...
+## New API leaks
+
+There are two ways to register new leaks:
+
+* Automatic - Wallarm actively scans thousands of public repositories and sources and adds new leaks to the list. Sort by **Status** and view `Opened` leaks - they require your attention.
+* Manual - add API leaks manually. Each one is a set of leaked tokens.
+
+![API Leaks - Manual adding](../images/about-wallarm-waf/api-leaks/api-leaks-add-manually.png)
+
+## Interactive visualization
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-leaks.md:46]: 2 sections about visaalization

File: docs/latest/about-wallarm/api-leaks.md [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
 
-For information on how to use the **API Leaks** module, refer to its [user guide](../user-guides/api-leaks.md).
+The module can operate in two different modes: 
+
+* **Only detect** API leaks and report that to users via the **API Leaks** section in Wallarm Cloud.
+* Detect and [**perform measures to block**](#making-decisions) usage of leaked credentials until they are regenerated or removed. This requires deployed Wallarm [node(s)](../user-guides/nodes/nodes.md).
 
 ![API Leaks](../images/about-wallarm-waf/api-leaks/api-leaks.png)
 
+By default, the API Leaks module is disabled. To get access to the module, please send a request to [Wallarm technical support](mailto:support@wallarm.com).
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-leaks.md:12]: it is difficult to notice this block

File: docs/latest/about-wallarm/api-leaks.md [hunk 5]
```diff
@@ -27,7 +32,51 @@ The **API Leaks** section provides rich visual representation for your current s
 
 ![API Leaks - Visualization](../images/about-wallarm-waf/api-leaks/api-leaks-visual.png)
 
-## Access API Leaks
+## New API leaks
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-leaks.md:35]: I suppose that visualization is smth hapenning after adding or detecting new leaks

File: docs/latest/about-wallarm/api-leaks.md [hunk 6]
```diff
@@ -27,7 +32,51 @@ The **API Leaks** section provides rich visual representation for your current s
...
+
+There are two ways to register new leaks:
+
+* Automatic - Wallarm actively scans the selected public repositories and sources and adds new leaks to the list. Sort by **Status** and view `Opened` leaks - they require your attention.
+* Manual - add API leaks manually. Each one is a set of leaked tokens.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-leaks.md:40]: a set of? how to separate several values then?

File: docs/4.8/about-wallarm/attack-surface.md [hunk 7]
```diff
@@ -0,0 +1 @@
+--8<-- "latest/about-wallarm/attack-surface.md"
```
Comment (AnastasiaTWW) [inline docs/4.8/about-wallarm/attack-surface.md:1]: home page is not updated

File: docs/latest/about-wallarm/api-leaks.md (hunk 6 above)
Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-leaks.md:40]: It is organized by UI: you click "New token" - the new field appears and you add the next token. It is displayed on the screenshot.

File: docs/3.6/user-guides/api-leaks.md [hunk 8]
```diff
@@ -1 +1,48 @@
...
+* Automatic - Wallarm actively scans thousands of public repositories and sources and adds new leaks to the list. Sort by **Status** and view `Opened` leaks - they require your attention.
+* Manual - add API leaks manually. Each one is a set of leaked tokens.
+
+![API Leaks - Manual adding](../images/about-wallarm-waf/api-leaks/api-leaks-add-manually.png)
+
+## Interactive visualization
+
+The **API Leaks** section provides rich visual representation for your current situation regarding found API leaks. Use the graphics to quickly analyze current situation with found leaks, click diagram elements to filter leaks by risk levels and sources.
+
+![API Leaks - Visualization](../images/about-wallarm-waf/api-leaks/api-leaks-visual.png)
```
Comment (AnastasiaTWW) [inline docs/3.6/user-guides/api-leaks.md:28]: use the more detailed screenshot. why this one? visalization is not only diagrams

File: docs/latest/user-guides/ip-lists/overview.md [hunk 9]
```diff
@@ -33,5 +33,4 @@ To configure IP lists:
 !!! warning "Using additional traffic filtering facilities"
     Note that if you use additional facilities (software or hardware) to automatically filter and block traffic, it is recommended that you configure an allowlist with the IP addresses for the [Wallarm Scanner](../../about-wallarm/detecting-vulnerabilities.md#vulnerability-scanner). This will allow Wallarm components to seamlessly scan your resources for vulnerabilities.
 
-    * [Scanner IP address registered in Wallarm US Cloud](../../admin-en/scanner-address-us-cloud.md)
-    * [Scanner IP address registered in Wallarm EU Cloud](../../admin-en/scanner-address-eu-cloud.md)
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/ip-lists/overview.md:37]: users view these pages, why dod we delete them?

Comment (AnastasiaTWW) [reply docs/latest/user-guides/ip-lists/overview.md:37]: I see what changes. add redirections

File: docs/4.8/about-wallarm/attack-surface.md (hunk 7 above)
Comment (AnastasiaTWW) [inline docs/4.8/about-wallarm/attack-surface.md:1]: all deleted pages must be accompanied with thre redirections

**Pull request merged**